tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c

```diff
@@ -145,6 +145,25 @@ static void start_per_adv_set(struct bt_le_ext_adv *adv)
 	printk("done.\n");
 }
 
+#if (CONFIG_BT_CTLR_ADV_DATA_CHAIN)
+static void set_per_adv_data(struct bt_le_ext_adv *adv)
+{
+	int err;
+	const struct bt_data ad[] = {
+		BT_DATA(BT_DATA_MANUFACTURER_DATA, mfg_data, 254),
+		BT_DATA(BT_DATA_MANUFACTURER_DATA, mfg_data, 254)};
```

Comment:
Is there any issue with using `BT_DATA_BYTES` on an `extern`? Otherwise I suggest to do
```suggestion
		BT_DATA_BYTES(BT_DATA_MANUFACTURER_DATA, mfg_data),
		BT_DATA_BYTES(BT_DATA_MANUFACTURER_DATA, mfg_data)};
```

If that is not possible, I suppose `sizeof(mfg_data)` also won't work instead of hardcoding the size to `254`

Reply:
Sorry I replied to one of the older comments regarding this but yes: 
I was getting build errors with BT_DATA_BYTES as it was expecting a char/uint8_t not a uint8_t *
Looking at other instances where BT_DATA_BYTES is used the data bytes are inline in the function and not a pointer to an array of the data. So have left it as BT_DATA 

But yeah using sizeof(mfg_data) sounds good

Reply:
should this also use ARRAY_SIZE instead of sizeof 
re: your other comment?